packages/account-management/src/frontend/Registration/Registration.tsx

```diff
@@ -147,8 +152,8 @@ export const Registration: React.FC = () => {
               label="First name"
               value={firstName}
               setValue={(value: string) => setFirstName(value)}
-              isValid={!!(firstName && firstName !== '')}
-              showValidity={!!valid}
+              isValid={firstName !== ''}
+              showValidity={!!firstName && firstName !== ''}
```

Comment:
Could we use a variable to avoid repetition here?

```ts
const isFirstNameValid = Boolean(firstName && firstName !== '');

// ...

  isValid={isFirstNameValid}
  showValidity={isFirstNameValid}
```